docs/linux/sql-server-linux-active-directory-authentication.md

```diff
@@ -41,6 +41,8 @@ Before you configure AD Authentication, you need to:
   * [Red Hat Enterprise Linux](quickstart-install-connect-red-hat.md)
   * [SUSE Linux Enterprise Server](quickstart-install-connect-suse.md)
   * [Ubuntu](quickstart-install-connect-ubuntu.md)
+  
+The output of the hostname command on the Linux server should be the fully qualified domain name of the server, resolvable in DNS, and should be used for all service definitions, E.G: MSSQLSvc/**<fully qualified domain name of host machine>**:**<tcp port>**
```

Comment:
This is confusing to have here, and there are no checks or instructions on how to fix the issue. Perhaps add a new subsection with steps for this